### PR TITLE
Updating and cleaning npm packages, updating the webpack build, updat…

### PR DESCRIPTION
**Pull request by EdwinGuzman**
…ing the readme, adding unit tests, coverage, and an example.

### CODE REVIEW

File: .nyc_output/28b6eec849100fb8405b1db8caf0bed3.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (rhernand3z) [inline .nyc_output/28b6eec849100fb8405b1db8caf0bed3.json]: Seems like this wasn't gitignored..

File: package.json [hunk 2]
```diff
@@ -81,11 +85,19 @@
     "react-tools": "0.13.3",
     "react-addons-css-transition-group": "0.14.8",
```
Comment (rhernand3z) [inline package.json]: Wouldn't this dependency conflict with the current react v15.x?

Comment (EdwinGuzman) [reply package.json]: Doesn't conflict and it's a dependency I hope we can remove soon.

File: .nyc_output/28b6eec849100fb8405b1db8caf0bed3.json (hunk 1 above)
Comment (EdwinGuzman) [reply .nyc_output/28b6eec849100fb8405b1db8caf0bed3.json]: Updated.

**Pull request merged**